### PR TITLE
Cookies banner privacy policy page

### LINKED ISSUE #469: Cookies Banner and Privacy Policy Page
### Overview

As a developers, We want to implement a Cookies Banner and Privacy Policy Page for compliance with data protection regulations and user transparency. For this issue we will create a cookies banner that will appear at the bottom of the website's pages. The banner should not obscure content and be dismissible by users. Additionally we will create a comprehensive Privacy Policy page that details our data handling practices, including data collection, storage, processing, and protection.

### Action Items

In accordance with the designs in Figma

- [x] Create Cookies Banner Component
- [x] Implement functionality for user consent regarding cookies
- [x] Create Privacy Policy Page 

### Resources/Instructions

[Resources](https://hackforla.github.io/CivicTechJobs/resources/)

### PR DESCRIPTION
**Pull request by bennyv8**
Fixes #469 

### Changes

- Implemented Privacy Policy Page
    - Added background and illustration asset
    - Utilized a CSS file to abstract repetitive tailwind classes *see notes*
    - *Note*: utilizing a domain name place holder
- Linked privacy policy to cookie banner
- Updated text-coloring in Cookie Banner Title to charcoal

### Notes

- The background.svg has a white border so when I scale it to fit the container, there is a slight white border on the left and right side. When I don't scale it to container, the borders are gone but it won't load dynamically to screen size. Unsure of what to do.

- The css file is not passing the linter because the @tailwind & @apply directive is unkown to prettier and vscode. I could either

    1. re-implement the css classes with **const className : string =...** in the privacyPolicyPage and remove the css file 
    2. Ignore the unknown rule for the CSS file in stylelint.rc to address prettier linter and follow [this](https://duncanleung.com/tailwind-css-unknown-at-rule/) to address the VScode warning

If there wouldn't be another instance where a CSS file to abstract tailwind classes is better, then I think re-factoring to the first option could be better. Let me know what y'all think

### Screenshots, if applicable

<details>
  <summary>Privacy Page</summary>
  <br>
  
  <br>
</details>

### CODE REVIEW

**Review by MattPereira — APPROVED** [review]
Well done @bennyv8 !!! :ok_hand: 

Great job communicating with the team during our last meeting and excellent work creating the issue and explaining your thought process in the PR creation comment

Keep up the good work! :tada:

File: frontend/tailwind.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     "./src/pages/Authentication/*.tsx",
     "./src/tw-components/*.tsx",
     "./src/pages/LandingPage/*.tsx",
+    "./src/pages/PrivacyPolicyPage/*"
```
Comment (MattPereira) [inline frontend/tailwind.config.js:8]: Looks like the linter is still angry this time about Prettier

My guess is it wants that trailing comma on the last item of the `content` array here like it had before

image

Comment (bzzz-coding) [reply frontend/tailwind.config.js:8]: I agree with @MattPereira I ran prettier on this file and it added a trailing comma here.

**Review by bzzz-coding — APPROVED** [review]
Thanks for working on this issue, @bennyv8 ! The privacy policy page looks great ✨🚀

**Pull request merged**